pytensor/link/pytorch/dispatch/elemwise.py

```diff
@@ -40,11 +40,14 @@ def dimshuffle(x):
 @pytorch_funcify.register(Softmax)
 def pytorch_funcify_Softmax(op, **kwargs):
     axis = op.axis
+    dtype = kwargs["node"].outputs[0].dtype
```

Comment:
Maybe better to check the input dtype? Sorry if I said the output before
```suggestion
    dtype = kwargs["node"].inputs[0].dtype
```